js/PulsarFormComponent.js

```diff
@@ -85,10 +89,30 @@ PulsarFormComponent.prototype.initTimePickers = function () {
 PulsarFormComponent.prototype.initDatePickers = function () {
     // Attach basic pikaday to datepicker fields
     this.$html
-        .find('[data-datepicker=true]')
+        .find('[data-datepickerStyle=default]')
         .pikaday({ format: 'DD/MM/YYYY' });
 }
 
+/**
+ * Initiate a date picker on data-datepicker fields using pickaday US Format
+ */
+PulsarFormComponent.prototype.initUSDatePickers = function () {
```

Comment:
if we accepted a format string we wouldn't need these separate initialisers, we could just pass the string straight to pikaday's format...